src/network/bmobject.py

```diff
@@ -1,44 +1,68 @@
-from binascii import hexlify
+"""
+src/network/bmobject.py
+======================
+
+"""
+
 import time
 
+import protocol
+import state
 from addresses import calculateInventoryHash
 from debug import logger
 from inventory import Inventory
 from network.dandelion import Dandelion
-import protocol
-import state
+
 
 class BMObjectInsufficientPOWError(Exception):
+    """"""
     errorCodes = ("Insufficient proof of work")
 
 
 class BMObjectInvalidDataError(Exception):
+    """"""
     errorCodes = ("Data invalid")
 
 
 class BMObjectExpiredError(Exception):
+    """"""
     errorCodes = ("Object expired")
 
 
 class BMObjectUnwantedStreamError(Exception):
```

Comment:
Exception indicating the object is in a stream we didn't advertise as being interested in.